calico_node/startup.py

```diff
@@ -266,7 +266,11 @@ def main():
         sys.exit(1)
 
     as_num = os.getenv("AS")
-    as_num = as_num or None
+    if os.getenv("USE_HOST_AS") == "true":
```

Comment:
@robbrockbank do you mean switch to the single environment variable version @caseydavenport suggested?